pkg/proxy/iptables/proxier.go

```diff
@@ -764,7 +806,7 @@ func getLocalIPs(endpointsMap proxyEndpointsMap) map[types.NamespacedName]sets.S
 				if localIPs[nsn] == nil {
 					localIPs[nsn] = sets.NewString()
 				}
-				ip := strings.Split(ep.endpoint, ":")[0] // just the IP part
+				ip := ep.endpoint[0:strings.Index(ep.endpoint, ":")] //just the IP part
```

Comment:
@thockin - should we guard? This is our internal data, so I think we can assume it's correct.
Yes we can store the splitted version too.